src/qt/optionsdialog.cpp

```diff
@@ -108,7 +110,12 @@ OptionsDialog::OptionsDialog(QWidget *parent, bool enableWallet) :
             ui->lang->addItem(locale.nativeLanguageName() + QString(" (") + langStr + QString(")"), QVariant(langStr));
         }
     }
+#ifdef ENABLE_WALLET
```

Comment:
Is this change even necessary? I think the above `(!enableWallet)` change covers all? if so, it's good to avoid introducing conditional compilation if possible.

Reply:
I have tested it and yes, it doesn't need this #ifdef. Many thanks!